Makefile

```diff
@@ -140,31 +140,5 @@ else
 	$(error Please run `go fmt ./...` to format the code)
 endif
 
-.PHONY: impi
-impi:
-	@echo Installing impi...
-	GO111MODULE=off go get -u github.com/pavius/impi/cmd/impi
-	@echo Verifying imports...
-	$(GOPATH)/bin/impi \
-		--local github.com/iguazio/provazio \
-		--skip pkg/controller/apis \
-		--skip pkg/controller/client \
-		--ignore-generated \
-		--scheme stdLocalThirdParty \
-		./...
-
-$(GOPATH)/bin/golangci-lint:
-	@echo Installing golangci-lint...
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.49.0
-	cp ./bin/golangci-lint $(GOPATH)/bin/
```

Comment:
Deleted it because it breaks in CI and takes forever to run locally. It's probably no worth our time to get it to work with latest golang.